src/modules/organization/organization.controller.e2e.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import {
+  getRepositoryToken,
+  TypeOrmModule,
+  TypeOrmModuleOptions,
+} from '@nestjs/typeorm';
+import { OrganizationController } from './organization.controller';
+import * as TestDbCOnfig from '../../../test/config';
+import { Organization } from './organization.entity';
+import { LoggerModule } from '../logger/logger.module';
+import { Application } from '../application/application.entity';
+import { SentryModule } from '../sentry/sentry.module';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { OrganizationService } from './organization.service';
+import { Chance } from 'chance';
+import { Connection, EntityManager, QueryRunner, Repository } from 'typeorm';
+import request from 'supertest';
+import { INestApplication } from '@nestjs/common';
+
+const chance = new Chance();
+
+const organizationFixture = async (
+  repo: Repository<Organization>,
+  count = 1,
+) => {
+  const organizations = [];
+  for (let i = 0; i < count; i++) {
+    const name = chance
+      .string({ pool: 'abcdefghijklmnopqrstuvwxyz' })
+      .toLowerCase();
+    const definition = {
+      orgName: name,
+      description: chance.paragraph(),
+      websiteUrl: chance.url(),
+    };
+    const org = Organization.create({
+      name,
+      namespace: `${name}.iam.ewc`,
+      owner: '0x7dD4cF86e6f143300C4550220c4eD66690a655fc',
+      definition,
+    });
+    organizations.push(org);
+  }
+  return repo.save(organizations);
+};
+
+describe('OrganizationController', () => {
+  let module: TestingModule;
+  let repo: Repository<Organization>;
+  let queryRunner: QueryRunner;
+  let testHttpServer: request.SuperTest<request.Test>;
+  let app: INestApplication;
+  let organizations: Organization[];
+
+  beforeEach(async () => {
+    module = await Test.createTestingModule({
+      imports: [
+        TypeOrmModule.forRoot(TestDbCOnfig.default as TypeOrmModuleOptions),
+        TypeOrmModule.forFeature([Organization, Application]),
+        LoggerModule,
+        ConfigModule,
+        SentryModule,
```

Comment:
the `SentryModule` needs to be injected here because of the way dependency injection works. The `OrganizationModule` imports `OrganizationService` which then imports `LoggerService` which then goes on to use `SentryErrorInterceptor` which would not be available and would throw an error if the `SentryModule` is not imported in the current `TestingModule` context so it might not be needed in the other e2e tests depending or wether they use  the LoggerService or not.